libs/utils/decorate.js

```diff
@@ -325,11 +386,23 @@ export async function loadCDT(el, classList) {
   }
 }
 
+export function isAccessible(anchorTag) {
+  return !anchorTag.hash.includes('hide-controls');
+}
+
 export function decorateAnchorVideo({ src = '', anchorTag }) {
   if (!src.length || !(anchorTag instanceof HTMLElement)) return;
+  const accessibilityEnabled = isAccessible(anchorTag);
+  anchorTag.hash = anchorTag.hash.replace('#hide-controls', '');
```

Comment:
Could we use the same `#_` pattern here as well? Since we have `#_autoplay`, `#_hoverplay` and so on, we should keep things consistent. It might also be a good idea to remove the anchor in the function itself so the logic stays grouped together.